### PR TITLE
Pages redesign

### PR DESCRIPTION
**Pull request by zpoken**

### CODE REVIEW

File: apps/extension/tailwind.config.js [hunk 1]
```diff
@@ -1 +1 @@
-module.exports = require('ui/tailwind.config');
+module.exports = { ...require('ui/tailwind.config') };
```
Comment (grod220) [inline apps/extension/tailwind.config.js:1]: Don't think this spread is necessary

File: packages/ui/styles/globals.css [hunk 2]
```diff
@@ -47,63 +47,58 @@
   #popup-root {
     width: 400px;
     height: 600px;
+    /* overflow-y: auto; */
```
Comment (grod220) [inline packages/ui/styles/globals.css:50]: remove comment

File: packages/ui/styles/globals.css [hunk 3]
```diff
@@ -47,63 +47,58 @@
...
   }
 
-  @font-face {
-    font-family: 'Faktum Bold';
-    src:
-      local('Faktum Bold'),
-      url('../fonts/Faktum-Bold.ttf') format('truetype');
+  /* Handle on hover */
+  ::-webkit-scrollbar-thumb:hover {
+    background: #555;
```
Comment (grod220) [inline packages/ui/styles/globals.css:70]: What do these scrollbar classes do? Why do we need them?

**Review by grod220 — COMMENTED** [review]
If you are not able to make these design changes, please work with Daria to make the fixes and apply them in this PR.

File: packages/ui/styles/globals.css (hunk 3 above)
Comment (zpoken) [reply packages/ui/styles/globals.css:70]: Знімок екрана 2023-09-18 о 14 18 30
we have scrool here, I decided to change the style

File: apps/extension/src/routes/page/onboarding/success.tsx [hunk 4]
```diff
@@ -6,23 +6,23 @@ import {
...
         </CardHeader>
-        <CardContent className='grid gap-4'>
-          <p>You are all set!</p>
-          <p>
+        <CardContent className='mt-2 grid gap-1 text-lg_bold'>
+          <p className='text-center'>You are all set!</p>
+          <p className='text-center text-muted-foreground'>
             Use your account to transact, stake, swap or market make. All of it is shielded and
             private.
           </p>
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/page/onboarding/success.tsx:28]: https://www.figma.com/file/iTnIhPxUm4ZleTRKWEn4GM/Penumbra?node-id=2220%3A3945&mode=dev 
Why didn't you leave a comment here?

File: apps/extension/src/routes/popup/settings/settings-fvk.tsx [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+        <form
+          className='flex flex-1 flex-col items-start justify-between px-[30px] pb-[30px]'
+          onSubmit={submit}
+        >
+          <div className='flex flex-col gap-4'>
+            {!fvk ? (
+              <PasswordInput
+                passwordValue={input}
+                label={
+                  <p className='font-headline text-xl_semiBold text-muted-foreground'>Password</p>
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/popup/settings/settings-fvk.tsx:46]: https://www.figma.com/file/iTnIhPxUm4ZleTRKWEn4GM/Penumbra?node-id=2218%3A3853&mode=dev
Why didn't you leave a comment here?

File: apps/extension/src/routes/page/onboarding/generate.tsx [hunk 6]
```diff
@@ -36,54 +35,53 @@ export const GenerateSeedPhrase = () => {
...
+              className={cn(
+                'grid gap-4 mt-2',
+                phrase.length === 12 ? 'grid-cols-3' : 'grid-cols-4',
+              )}
+            >
               {phrase.map((word, i) => (
                 <div className='flex flex-row items-center justify-center gap-2' key={i}>
-                  <div className='w-7 text-right'>{i + 1}.</div>
-                  <Input readOnly value={word} />
+                  <div className='w-7 text-right font-headline text-lg_medium'>{i + 1}.</div>
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/page/onboarding/generate.tsx:54]: https://www.figma.com/file/iTnIhPxUm4ZleTRKWEn4GM/Penumbra?node-id=2208%3A3443&mode=dev

File: apps/extension/src/routes/page/onboarding/start.tsx [hunk 7]
```diff
@@ -7,25 +7,25 @@ import {
...
 
 export const OnboardingStart = () => {
   const navigate = usePageNav();
 
   return (
     <FadeTransition>
-      <CompressedVideoLogo className='mb-12 self-center' />
-      <Card className='w-[450px] p-6' gradient>
+      <CompressedVideoLogo className='mb-10 self-center' />
+      <Card className='w-[440px]' gradient>
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/page/onboarding/start.tsx:20]: https://www.figma.com/file/iTnIhPxUm4ZleTRKWEn4GM/Penumbra?node-id=2189%3A3138&mode=dev

File: apps/extension/src/shared/components/settings-header.tsx [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+export const SettingsHeader = ({ title }: { title: string }) => {
+  const navigate = usePopupNav();
+  return (
+    <>
+      <BackIcon
+        className='absolute ml-[30px] mt-6 h-6 w-6 text-muted-foreground'
+        onClick={() => navigate(-1)}
+      />
+      <h1 className='h-[60px] border-b border-[rgba(75,75,75,0.50)] pb-2 pt-5 text-center font-headline text-3xl'>
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/shared/components/settings-header.tsx:12]: https://www.figma.com/file/iTnIhPxUm4ZleTRKWEn4GM/Penumbra?node-id=2508%3A5192&mode=dev

File: apps/extension/src/routes/popup/settings/settings-connected-sites.tsx [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+                    <DotsVerticalIcon className='h-5 w-5 cursor-pointer hover:opacity-50' />
+                  </PopoverTrigger>
+                  <PopoverContent align='center' className='w-[120px] p-0 pb-3'>
+                    <Button variant='outline' className='flex h-11 w-full justify-start px-5'>
+                      Delete
+                    </Button>
+                    <Button variant='outline' className='flex h-11 w-full justify-start px-5'>
+                      Edit
+                    </Button>
+                  </PopoverContent>
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/popup/settings/settings-connected-sites.tsx:53]: https://www.figma.com/file/iTnIhPxUm4ZleTRKWEn4GM/Penumbra?node-id=2426%3A7377&mode=dev

File: apps/extension/src/routes/popup/settings/settings.tsx [hunk 10]
```diff
@@ -1,55 +1,73 @@
...
-            className='flex w-full items-center justify-start gap-2 p-[10px] text-left hover:bg-transparent hover:opacity-50'
-            onClick={() => navigate(i.href)}
-          >
-            {i.icon}
-            <p className='text-foreground'>{i.title}</p>
-          </Button>
-        ))}
+    <FadeTransition>
+      <div className='flex min-h-[100vh] w-[100vw] flex-col justify-between gap-6'>
+        <SettingsHeader title='Settings' />
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/popup/settings/settings.tsx:50]: https://www.figma.com/file/iTnIhPxUm4ZleTRKWEn4GM/Penumbra?node-id=2508%3A5192&mode=dev

File: apps/extension/src/routes/popup/home/index.tsx [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+            }
+            className='w-4'
+          />
+        </div>
+      </div>
+      <div className='flex flex-col items-center gap-1'>
+        <p className='font-headline text-3xl text-sand'>Syncing blocks...</p>
+        <p className='font-headline text-[22px] font-medium leading-8 text-muted-foreground'>
+          10982/121312
+        </p>
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/popup/home/index.tsx:60]: https://www.figma.com/file/iTnIhPxUm4ZleTRKWEn4GM/Penumbra?node-id=2476%3A8101&mode=dev

File: apps/extension/src/routes/popup/settings/settings-security.tsx [hunk 12]
```diff
@@ -1,15 +1,207 @@
...
+        <div className='flex flex-1 flex-col items-start gap-2 px-[30px]'>
+          {links.map(i => (
+            <CustomLink
+              key={i.href}
+              title={i.title}
+              icon={i.icon}
+              onClick={() => navigate(i.href)}
+            />
+          ))}
+        </div>
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/popup/settings/settings-security.tsx:42]: https://www.figma.com/file/iTnIhPxUm4ZleTRKWEn4GM/Penumbra?node-id=2508%3A5580&mode=dev

File: apps/extension/src/routes/popup/accounts.tsx [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+import { PopupPath } from './paths';
+
+export const Accounts = () => {
+  const navigate = usePopupNav();
+  const { all } = useStore(walletsSelector);
+
+  return (
+    <FadeTransition>
+      <div className='flex min-h-[100vh] w-[100vw] flex-col justify-between gap-6'>
+        <SettingsHeader title='Select Wallet' />
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/popup/accounts.tsx:16]: we have it there - https://www.figma.com/file/iTnIhPxUm4ZleTRKWEn4GM/Penumbra?node-id=2508%3A5580&mode=dev. or rename it to wallet?

File: apps/extension/src/routes/page/index.tsx [hunk 14]
```diff
@@ -21,9 +20,6 @@ export const pageIndexLoader = async () => {
 
   if (!wallets.length) return redirect(PagePath.WELCOME);
 
-  const password = await sessionExtStorage.get('hashedPassword');
-  if (!password) return redirect(PagePath.LOGIN);
-
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/page/index.tsx:26]: we decided that our tab performs the function of creating an account and recovering a password, so there is no point in leaving the login page

File: apps/extension/src/routes/popup/settings/settings-clear-cache.tsx [hunk 15]
```diff
@@ -0,0 +1,83 @@
+import { ExclamationTriangleIcon } from '@radix-ui/react-icons';
+import { FadeTransition, SettingsHeader } from '../../../shared';
+import { Button } from 'ui/components';
+
+// There we shold clear indexedDB, balances; stop sync, logout
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/popup/settings/settings-clear-cache.tsx:5]: this is for the future, what actions to take when we clear the cache

File: apps/extension/src/routes/popup/settings/settings-auto-lock.tsx [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+          <Button variant='gradient' className='h-11 w-full'>
+            Save
+          </Button>
+        </div>
+      </div>
+    </FadeTransition>
+  );
+};
+
+const TimerGradientIcon = () => (
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/popup/settings/settings-auto-lock.tsx:29]: Знімок екрана 2023-09-18 о 17 38 43

Something like there?

File: apps/extension/src/routes/page/onboarding/generate.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/routes/page/onboarding/generate.tsx:54]: What is the feedback?

File: apps/extension/src/routes/page/onboarding/start.tsx (hunk 7 above)
[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/routes/page/onboarding/start.tsx:20]: That design has more padding than this react element

File: apps/extension/src/routes/page/onboarding/success.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/routes/page/onboarding/success.tsx:28]: The design has a much larger headline font

File: apps/extension/src/routes/popup/accounts.tsx (hunk 13 above)
[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/routes/popup/accounts.tsx:16]: Ah ok. That's fine. But the feedback on the first comment is still valid.

File: apps/extension/src/routes/popup/home/index.tsx (hunk 11 above)
[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/routes/popup/home/index.tsx:60]: Looks like the design is being adjusted?

File: apps/extension/src/routes/popup/settings/settings-auto-lock.tsx (hunk 16 above)
[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/routes/popup/settings/settings-auto-lock.tsx:29]: No, we should have a top level folder called "Icons" where they all live.

File: apps/extension/src/routes/popup/settings/settings-clear-cache.tsx (hunk 15 above)
[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/routes/popup/settings/settings-clear-cache.tsx:5]: Yes, I understand. It just needs to be written out more. The original comment is not clear enough.

File: apps/extension/src/routes/popup/settings/settings-connected-sites.tsx (hunk 9 above)
[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/routes/popup/settings/settings-connected-sites.tsx:53]: Looks like it's been fixed?

File: apps/extension/src/routes/popup/settings/settings-fvk.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/routes/popup/settings/settings-fvk.tsx:46]: Looks like it's been fixed?

File: apps/extension/src/routes/popup/settings/settings-security.tsx (hunk 12 above)
[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/routes/popup/settings/settings-security.tsx:42]: What is the feedback here?

File: apps/extension/src/routes/popup/settings/settings.tsx (hunk 10 above)
[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/routes/popup/settings/settings.tsx:50]: In the design it is centered

File: apps/extension/src/routes/popup/home/index.tsx [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+
+  if (!password) return redirect(PopupPath.LOGIN);
+
+  return null;
+};
+
+export const PopupIndex = () => {
+  const { all } = useStore(walletsSelector);
+  return (
+    <FadeTransition className='flex flex-col items-stretch justify-start px-7'>
```
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/popup/home/index.tsx:33]: can we create isLoading variable in global state?we will be able to display the loader if the data has not yet been downloaded from the storage

File: apps/extension/src/shared/components/settings-header.tsx (hunk 8 above)
[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/shared/components/settings-header.tsx:12]: It's too small and hard to read. We have plenty of space to make larger.

File: apps/extension/src/routes/popup/home/index.tsx (hunk 17 above)
[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/routes/popup/home/index.tsx:33]: The loading is so quick, we don't need to display a spinner. The feedback is to 1) remove the fading transition for this screen (and see if that helps) and 2) explore whether a react router loader may help in this situation or not.

File: apps/extension/src/routes/popup/settings/settings-fvk.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/popup/settings/settings-fvk.tsx:46]: yes

File: apps/extension/src/routes/popup/home/index.tsx (hunk 17 above)
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/popup/home/index.tsx:33]: Знімок екрана 2023-09-18 о 18 14 23

we will have more downloads in the future

File: apps/extension/src/routes/popup/settings/settings-auto-lock.tsx (hunk 16 above)
[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/popup/settings/settings-auto-lock.tsx:29]: /settings/icons?

[the remark this answers is not in the document]
Comment (zpoken) [reply apps/extension/src/routes/popup/settings/settings-auto-lock.tsx:29]: Знімок екрана 2023-09-18 о 18 17 50

or add it to share folder?

[the remark this answers is not in the document]
Comment (grod220) [reply apps/extension/src/routes/popup/settings/settings-auto-lock.tsx:29]: `/settings/icons` or even `/icons`(not in components) would be fine in my opinion

File: apps/extension/src/routes/popup/home/index.tsx [hunk 18]
```diff
@@ -0,0 +1,67 @@
...
+            className='w-4'
+          />
+        </div>
+      </div>
+      <div className='flex flex-col items-center gap-1'>
+        <p className='mb-1 font-headline text-xl font-semibold leading-[30px] text-sand'>
+          Syncing blocks...
+        </p>
+        <Progress value={73} />
+        <p className='text-2xl font-bold leading-[30px] text-sand'>10982/121312</p>
```
Comment (grod220) [inline apps/extension/src/routes/popup/home/index.tsx:63]: The font size should be the same as `syncing blocks`

File: apps/extension/src/routes/popup/settings/settings.tsx [hunk 19]
```diff
@@ -1,55 +1,73 @@
...
+        <div className='flex flex-1 flex-col items-start gap-5 px-[30px]'>
```
Comment (grod220) [inline apps/extension/src/routes/popup/settings/settings.tsx:51]: Thes icons need to be vertically centered with the icon next to it so they are the same height.

Screenshot 2023-09-20 at 2 30 44 PM

File: apps/extension/src/routes/page/onboarding/start.tsx
Comment (grod220) [inline apps/extension/src/routes/page/onboarding/start.tsx:1]: The button text should not be bold

Screenshot 2023-09-20 at 2 36 06 PM

File: apps/extension/src/shared/containers/word-length-toogles.tsx [hunk 20]
```diff
@@ -0,0 +1,28 @@
+import { SeedPhraseLength } from 'penumbra-crypto-ts/src/mnemonic';
+import { Toggle } from 'ui/components';
+
+interface WordLengthTooglesProsp {
+  toogleClick: (length: SeedPhraseLength) => void;
+  phrase: string[];
+}
+
+export const WordLengthToogles = ({ toogleClick, phrase }: WordLengthTooglesProsp) => {
```
Comment (grod220) [inline apps/extension/src/shared/containers/word-length-toogles.tsx:9]: If the toggle is selected, it should be white text. In this case `12 words` should be white.

Screenshot 2023-09-20 at 2 38 05 PM

File: apps/extension/src/routes/popup/account/account-popover.tsx [hunk 21]
```diff
@@ -0,0 +1,47 @@
...
+        <Button variant='outline' className='flex h-11 w-full justify-start px-5'>
+          Change Wallet Name
+        </Button>
+        <Button
+          variant='outline'
+          className='flex h-11 w-full justify-start rounded-b-lg border-b-0 px-5'
+        >
+          Delete Wallet
+        </Button>
+      </PopoverContent>
```
Comment (grod220) [inline apps/extension/src/routes/popup/account/account-popover.tsx:44]: These are no longer wallets. So the options really are only two:
- Change address label
- Remove address

**Review by grod220 — COMMENTED** [review]
A few more things. But we are close.

File: apps/extension/src/routes/page/onboarding/start.tsx
Comment (zpoken) [reply apps/extension/src/routes/page/onboarding/start.tsx:1]: for all button?

File: apps/extension/src/routes/popup/settings/settings.tsx (hunk 19 above)
Comment (zpoken) [reply apps/extension/src/routes/popup/settings/settings.tsx:51]: it`s not clear for me

Comment (grod220) [reply apps/extension/src/routes/popup/settings/settings.tsx:51]: Advanced it to high. I needs to be in the middle. It's the same for the rest as well.

269282671-66cddb54-f27c-46be-a143-4124fbc72dcf

File: apps/extension/src/routes/page/onboarding/start.tsx
Comment (grod220) [reply apps/extension/src/routes/page/onboarding/start.tsx:1]: Yes

**Review by grod220 — APPROVED** [review]

**Pull request merged**